apps/visual-test/src/App.tsx

```diff
@@ -0,0 +1,84 @@
+import { ChangeEvent, useState } from 'react';
+import FormMessageItemBody from '../../../src/ui/FormMessageItemBody';
+import { UserMessage } from '@sendbird/chat/message';
+import SendbirdChat from '@sendbird/chat';
+import { GroupChannelModule } from '@sendbird/chat/groupChannel';
+import { OpenChannelModule } from '@sendbird/chat/openChannel';
+import { mockUserMessage } from '../mock-data/form.ts';
+import { getMockMessageForm } from './utils.ts';
+// TODO: Need to change below imports into a single import later.
+import '../../../src/ui/TextMessageItemBody/index.scss';
+import '../../../src/ui/Input/index.scss';
+import '../../../src/ui/FormMessageItemBody/index.scss';
+
+
```

Comment:
Note that this testing app is currently implemented for Form testing only. However, this visual-test app is supposed to be for all visual tests not just form. For now lets just keep as is and later when we need to add another type of visual test other than form, we can refactor then.